react/utils/getFilters.js

```diff
@@ -74,4 +103,26 @@ const getFilters = ({
   ].filter(Boolean)
 }
 
+const getFormattedDeliveries = (deliveries, showShippingFacet) => {
+  if (!showShippingFacet) {
+    return deliveries.filter(d => d.name !== SHIPPING_KEY)
+  }
```

Comment:
Just checking, this is to support facets other than `shipping` in the future for Delivery Promises, correct?

Reply:
Correct, to ensure I'm not removing other facets that can be in this array